### ISSUE TITLE
Horizontal Scroll Bug

### ISSUE THREAD

**Issue by LyricalString** [MaTeMaTuK/gantt-task-react#70]
If I create a long gantt table (it does not happen with short ones under my experience), whenever I move fast enough the horizontal scroll, the scroll bugs and goes straight and back for a few seconds and it stops.

If I use the keys, then that bug stops happening for that instance, but whenever I move the scroll back, it happens.

**Comment by LyricalString** [comment]
https://user-images.githubusercontent.com/79110588/171828512-e35c07ac-3943-4722-abc2-0e60c73facce.mp4

**Comment by Alecattie** [comment]
Having the same issue

**Comment by MaTeMaTuK** [comment]
@LyricalString Could you please take the latest version and check?

**Comment by yzStrive** [comment]
@MaTeMaTuK  still have this problem

**Comment by yzStrive** [comment]
This problem persists, and the horizontal scrollbar feature of the simulation causes repeated rendering, causing performance problems and UI issues


This effect causes: https://github.com/MaTeMaTuK/gantt-task-react/blob/fabcdfb732e924df1126dbad3193e50a6b00cc1f/src/components/gantt/gantt.tsx#L258

**Comment by AlbajramiKevin** [comment]
> This problem persists, and the horizontal scrollbar feature of the simulation causes repeated rendering, causing performance problems and UI issues
> 
> This effect causes:
> 
> https://github.com/MaTeMaTuK/gantt-task-react/blob/fabcdfb732e924df1126dbad3193e50a6b00cc1f/src/components/gantt/gantt.tsx#L258

Did you find a solution for this?

**Comment by Kcamplone** [comment]
Anyone solved this issue ?

**Comment by luanlucho** [comment]
Issue still persists. Why is it closed? @MaTeMaTuK

**Comment by MarcosParengo** [comment]
I had a similar issue and noticed that React was logging a lot of errors to the console. 
![image](https://github.com/MaTeMaTuK/gantt-task-react/assets/51425159/68ef2b5d-22a4-4832-bb72-30f5cac83867)
I also experienced a strange scroll stuttering bug, like the one highlighted in the @LyricalString video.
After reviewing the examples in this repository and comparing configurations, I identified that I hadn't set the `progress` prop on some tasks. Once I assigned values of 0 or 100 to this prop, the errors ceased, and the scroll issue was resolved.

Hope this proves helpful to someone else :)
